omise-wc-gateway.php

```diff
@@ -199,10 +210,34 @@ public function process_payment($order_id) {
 					} else {
 						throw new Exception ( "Please select a card or enter new payment information." );
 					}
-									
-					$result = Omise::create_charge ( $this->private_key, $data );
-					$success = $this->is_charge_success($result);
+
+					$result = Omise::create_charge($this->private_key, $data);
+					if ($result->object === "error") {
+						throw new Exception ($result->message);
+					} if ($result->failure_code) {
+						throw new Exception ($result->failure_message);
+					} else {
+						$post_id = wp_insert_post(array(
+							'post_title' 	=> 'Omise Charge Id '.$result->id,
+							'post_type'		=> 'omise_charge_items',
```

Comment:
I'm not sure is it possible to correct an indent.
Wordpress have a standard for php to `Use real tabs and not spaces` but when we use the tab for make an indentation. It will be displayed incorrectly in Github.
ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

Do you have any suggestion about these ?
Thanks krub